bridge/client/app/_components/_dialogs/ktb-deletion-dialog/ktb-deletion-dialog.component.ts

```diff
@@ -44,6 +46,10 @@ export class KtbDeletionDialogComponent implements OnInit, OnDestroy {
   }
 
   public deleteConfirm(): void {
-    this.eventService.deletionTriggeredEvent.next({ type: this.data.type, name: this.data.name });
+    this.eventService.deletionTriggeredEvent.next({
+      type: this.data.type,
+      name: this.data.name,
+      context: this.data.context,
+    });
```

Comment:
Can be simplified to
```suggestion
    this.eventService.deletionTriggeredEvent.next(this.data);
```